src/game_config_manager.cpp

```diff
@@ -635,11 +635,11 @@ void game_config_manager::load_addons_cfg()
 		const std::string& msg1 =
 			_n("The following add-on had errors and could not be loaded:",
 			   "The following add-ons had errors and could not be loaded:",
-			   n);
+			   static_cast<int>(n));
```

Comment:
Passing the result of `<container>::size()` to `_n` seems a logical thing to do, I think we  should add an overload to src/gettext.hpp that accepts `std::size_t` instead of casting it in multiple places.

Reply:
So essentially:
```
[[maybe_unused]] inline static std::string _n(const char* str1, const char* str2, std::size_t n)
{ return translation::dsngettext(GETTEXT_DOMAIN, str1, str2, static_cast<int>(n)); }
```
?